x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java

```diff
@@ -374,6 +374,66 @@ public void testGroupingInsideGrouping() {
         );
     }
 
+    public void testInvalidBucketCalls() {
```

Comment:
Thank you! I think the validation for `BUCKET` is complex enough that we'll be very happy about this test when we make changes in the future.